DependencyInjection/Configuration.php

```diff
@@ -79,6 +79,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('title')->end()
                 ->scalarNode('description')->end()
                 ->scalarNode('original_route_pattern')->defaultValue(SeoPresentation::ORIGINAL_URL_CANONICAL)->end()
+                ->scalarNode('locale')->defaultNull()->end()
```

Comment:
Either `'en'` or `%locale%` or anything else you want.

However, your question reveals something even more important: `%locale%` doesn't hold the current locale. It just holds the default locale. If we want to *current* locale, we need to use `$request->getLocale()`.